src/Chapter1.hs

```diff
@@ -489,8 +489,9 @@ Implement a function that returns the last digit of a given number.
   results. Or you can try to guess the function name, search for it and check
   whether it works for you!
 -}
--- DON'T FORGET TO SPECIFY THE TYPE IN HERE
-lastDigit n = error "lastDigit: Not implemented!"
+lastDigit :: Int -> Int -- What about last digit of a float?
+lastDigit n = 
+    if n < 0 then mod (n * (-1)) 10 else mod n 10
```

Comment:
You can use `abs` function to get the absolute value of the number :) So you can skip the check on negative numbers 🙂